tests/lib/common.py

```diff
@@ -389,3 +392,64 @@ def configure_postfix(device):
     device.expect(device.prompt)
     device.sendline("service postfix reload")
     assert 0 != device.expect(['failed']+ device.prompt, timeout = 20) , "Unable to reolad server with new configurations"
+
+
+def file_open(file_path):
+    extension = os.path.splitext(file_path)[1]
+    with open(file_path) as f:
+        try:
+            if "json" in extension:
+                read_data = json.load(f)
+            elif "txt" in extension:
+                read_data = f.read()
+            return read_data
+        except Exception as e:
+            print "Could not read file from the path:", file_path
+            raise (e)
+
+def snmp_mib_set(device, board, iface_ip, mib_name, index, set_type, set_value, timeout=10, retry=3):
```

Comment:
These SNMP functions are OK- but they really could be in their own commit so we could let them through and not be held up by the other reviews.